website/blog/2018-01-18-implementing-twitters-app-loading-animation-in-react-native.md

```diff
@@ -286,6 +286,6 @@ This component has been published to npm and is on GitHub as [react-native-mask-
 
 ## More Reading / Extra Credit
 
-1. [This gitbook](http://browniefed.com/react-native-animation-book/) is a great resource to learn more about Animated after you have read the React Native docs.
+1. [This gitbook](https://browniefed.com/react-native-animation-book/) is a great resource to learn more about Animated after you have read the React Native docs.
```

Comment:
FYI: this link redirects to https://www.codedaily.io/courses/React-Native-Animated-for-Beginners